index.js

```diff
@@ -61,18 +73,21 @@ function createWrapFn(originalsToProxies, proxiesToOriginals, unwrapFn) {
         const privateHandler = typeof original === 'function'
             ? () => { }
             : {};
-        privateHandlersOriginals.set(privateHandler, original);
-        allHandlers.add(privateHandler);
+
+        // TODO
+        // privateHandlersOriginals.set(privateHandler, original);

```

Comment:
```suggestion
        privateHandlersOriginals.set(privateHandler, original);
        allHandlers.add(privateHandler);
```

Reply:
This isn't solved yet: I had it set to `new Proxy`, not `newProxy`. I'll add an explicit comment for that.